R/plots-qc.R

```diff
@@ -172,57 +172,26 @@ qc_cor_heatmap <- function(gimap_dataset) {
 #' Assumes the first column of the dataset is the day0 data; do I need a better
 #' method to tell, especially if there are reps?
 #' @param gimap_dataset The special gimap_dataset from the `setup_data` function which contains the transformed data
-#' @param cutoff default is NULL, the cutoff for low log2 CPM values for the plasmid time period
+#' @param cutoff default is NULL, the cutoff for low log2 CPM values for the plasmid time period; if not specified, The lower outlier (defined by taking the difference of the lower quartile and 1.5 * interquartile range) is used
 #' @param wide_ar aspect ratio, default is 0.75
 #' @importFrom magrittr %>%
 #' @import ggplot2
-#' @return a named list
+#' @return a ggplot histogram
 
 qc_plasmid_histogram <- function(gimap_dataset, cutoff = NULL, wide_ar = 0.75) {
   to_plot <- data.frame(gimap_dataset$transformed_data$log2_cpm[, 1]) %>% `colnames<-`(c("log2_cpm"))
```

Comment:
```suggestion
  to_plot <- data.frame(gimap_dataset$transformed_data$log2_cpm[, 1]) %>% `colnames<-`(c("log2_cpm"))
```

I think this is supposed to be grabbing the column of the Day 0 plasmids? So can we make this argument so that it works similarly to the other one where we specify columns? I think that would work just in case people don't have the first column as this plasmid data.

Reply:
I think that functionality is added in PR #35 (which is stacked on top of this one)